### PR TITLE
Standalone vue + Composable tracking + vue-i18n + storybook fixes + dependencies upgrade

### PR DESCRIPTION
**Pull request by antoinemetifeu**
# Issues
- [ACCOUNT-1081](https://forge.prestashop.com/browse/ACCOUNT-1081) (Standalone Vuejs)
- [ACCOUNT-910](https://forge.prestashop.com/browse/ACCOUNT-910) (Composable tracking - Should fix ps_google storybook)
- [ACCOUNT-996](https://forge.prestashop.com/browse/ACCOUNT-996) (Vue-i18n)
- [ACCOUNT-1080](https://forge.prestashop.com/browse/ACCOUNT-1080) (Fix storybook + upgrade dependencies)

### CODE REVIEW

**Comment by atourneriePresta** [comment]
To use Vue 3 as cdn and instanciate accounts_vue_components in it:

`<script type="module">
    import { createApp } from 'https://unpkg.com/vue@3.2.26/dist/vue.esm-browser.js';

    const App = {
        data() {
            return {
                name: "Anthony",
            };
        },
        template: `<h1>Hello {{ name }}</h1>`,
    };

    createApp(App).mount("#app3");
</script>`

And after instance your vue_components as you usually do ;)

File: public/index.html [hunk 1]
```diff
@@ -13,24 +13,13 @@
     </noscript>
     
     <div class="m-4">
-      <div id="app"></div>
+      <prestashop-accounts>
+          test slot
+          <!--<event-bus-not-installed />-->
+          <template v-slot:footer><span style="background: red">test footer</span></template>
```
Comment (aAmorim27) [inline public/index.html:19]: Maybe we need to clean these ?

**Review by aAmorim27 — APPROVED** [review]
Ok for this PR, it just need a bit of cleaning and maybe some docs about how to implement it in a new project ? :)

File: public/index.html (hunk 1 above)
Comment (atourneriePresta) [reply public/index.html]: Yes it needs to be clean

**Review by aAmorim27 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -49,12 +49,53 @@ https://storybook-accounts.distribution.prestashop.net/
...
+    window.psaccountsVue.init();
+</script>
+```
+
+If you need to use the vue cdn for your app, please create your app with the vue esm-browser as follow
+
+```
+<div id="app"></div>
+
+<script src="http://prestashop17.atournerie.docker.localhost/upload/psaccountsVue.umd.min.js"></script>
```
[the remark this answers is not in the document]
Comment (atourneriePresta) [reply README.md]: I'm sure you right ^^

**Pull request merged**